tests/unittests/test_util.py

```diff
@@ -341,6 +341,19 @@
 """
 )
 
+OS_RELEASE_MARINER = dedent(
```

Comment:
To compare, here is the os-release for the latest release.  Perhaps worth updating?
```
NAME="Common Base Linux Mariner"
VERSION="2.0.20221004"
ID=mariner
VERSION_ID="2.0"
PRETTY_NAME="CBL-Mariner/Linux"
ANSI_COLOR="1;34"
HOME_URL="https://aka.ms/cbl-mariner"
BUG_REPORT_URL="https://aka.ms/cbl-mariner"
SUPPORT_URL="https://aka.ms/cbl-mariner"
```

Reply:
change as suggested